CHANGES.md

```diff
@@ -1,3 +1,9 @@
+# 2.1.0 (Unreleased)
+- [NEW] Added support for authenticating with IAM API keys. See
+  README.md and the
```

Comment:
may as well make this a link to the README for anyone reading the changes on GitHub.

Reply:
Done in 50dacd1e